x-pack/plugins/task_manager/server/lib/log_health_metrics.ts

```diff
@@ -80,7 +80,7 @@ export function logHealthMetrics(
     // This is legacy support - we used to always show this
     logger.debug(message);
     if (logLevel !== LogLevel.Debug && lastLogLevel === LogLevel.Debug) {
-      logger.warn(
+      logger.debug(
```

Comment:
Should we still log at different levels when enabling `monitored_stats_health_verbose_log.enabled` or should it all be debug logs as well?

Reply:
Ya, interesting.  So if you have that enabled, you'll see the health report messages at one of three levels, depending on TM's health - error, warn, or debug.  That seems fine to me - you would expect to see more when enabling that config.  And one thing that could be nice - you could enable that, but not verbose logging in general (or just for TM), and then you'd have a less-noisy log, with just error/warn messages for the health report, and the level tells us whether TM considered it an error or warn.

I think we'll have to see how this works out in practice, but seems close to right, to me, for now.

Reply:
Ah, that's a good way to think about it. +1 seems close to right, and we'll wait and see before changing anything 👍